tests/e2e-test.spec.js

```diff
@@ -93,27 +94,26 @@ describe('Main workflow Test Suite', () => {
     const pgTableContact = await client.query(`SELECT * from ${PGTABLE} where _id = $1`, [contact._id]);
     expect(pgTableContact.rows[0]._deleted).to.equal(true);
     await delay(6); // wait for DBT
-    const modelContactResult = await client.query(`SELECT * FROM ${pgSchema}.contacts where _id = $1`, [contact._id]);
-    expect(modelContactResult.rows.length).to.equal(1);
-    expect(modelContactResult.rows[0]._deleted).to.equal(true);
+    const modelContactResult = await client.query(`SELECT * FROM ${pgSchema}.contacts where uuid = $1`, [contact._id]);
+    expect(modelContactResult.rows.length).to.equal(0);
   });
 
   it('should process person deletes', async () => {
     const person = contacts().find(contact => contact.type === 'person' && !contact._deleted);
 
-    const preDelete = await client.query(`SELECT * FROM ${pgSchema}.persons where _id = $1`, [person._id]);
+    const preDelete = await client.query(`SELECT * FROM ${pgSchema}.persons where uuid = $1`, [person._id]);
     expect(preDelete.rows.length).to.equal(1);
 
     await deleteDoc(person);
     await delay(6); // wait for CHT-Sync
     await delay(6); // wait for DBT
     await delay(6); // wait for DBT
 
-    const modelContactResult = await client.query(`SELECT * FROM ${pgSchema}.contacts where _id = $1`, [person._id]);
-    expect(modelContactResult.rows[0]._deleted).to.equal(true);
```

Comment:
Does the contact row get deleted at some point?

Reply:
Yes, we use a `post_hook` to [delete](https://github.com/medic/cht-sync/blob/e4408e1f3a8da2be3608c1f10bcdd3733d6a8785/tests/dbt/package/models/contacts/contacts.sql#L6-L7) any rows with the `_deleted` flag set to true

Reply:
How can I see this when testing?